tests/MonologCreator/Factory/HandlerTest.php

```diff
@@ -328,32 +321,23 @@ public function testCreateRedis()
             '{
                 "handler" : {
                     "redis" : {
-                        "url" : "mockUrl",
                         "key" : "mockKey"
                     }
                 }
             }',
             true
         );
 
-        $factory = $this->getMockBuilder('\MonologCreator\Factory\Handler')
-            ->setConstructorArgs(
-                [
-                    $config,
-                    array(
-                        'INFO' => Monolog\Logger::INFO,
-                    ),
-                    $this->mockFormatterFactory,
-                ]
-            )
-            ->setMethods(['createPredisClient'])
-            ->getMock();
-
-        $factory->expects($this->exactly(1))
-            ->method('createPredisClient')
-            ->with($this->equalTo('mockUrl'))
-            ->will($this->returnValue($this->mockPredisClient));
+        $factory      = new Handler(
+            $config,
+            array(
+                'INFO' => Monolog\Level::Info,
+            ),
+            $this->mockFormatterFactory,
+            new \Predis\Client('')
```

Comment:
Shouldn't this be a mock?

Reply:
Why? In this case only the creation of the Redishandler will be tested and so far I know the predis object itself is only unsed, once a log message is processed.

Reply:
Hm fair enough. It was just my initial reaction that normally in a unit test I would expect only mocks apart from the object to test.